framework/src/media/Decoder.cpp

```diff
@@ -136,6 +136,11 @@ bool Decoder::getFrame(unsigned char *buf, size_t *size, unsigned int *sampleRat
 		return false;
 	}
 
+	if (mChannels != *channels) {
```

Comment:
Please Add Description in decode.h regarding mChannels
For example,
TizenRT Media Framework Supports only ....blahblahblah... will be changed to stereo forcely.....blahblah

Reply:
fixed, thanks